lib/thinreports/generator/pdf/document/graphics/text.rb

```diff
@@ -27,9 +27,9 @@ def text_box(content, x, y, w, h, attrs = {})
         box_attrs = text_box_attrs(x, y, w, h, single: attrs.delete(:single),
                                                overflow: attrs[:overflow])
         # Do not break by word unless :word_wrap is :break_word
-        content = text_without_line_wrap(content) if attrs[:word_wrap] == :none
 
         with_text_styles(attrs) do |built_attrs, font_styles|
+          content = text_without_line_wrap(content) if attrs[:word_wrap] == :none && pdf.font.unicode?
```

Comment:
Oh I get it, but i think that we need to clarity behavior of `attrs[:word_wrap] == :none` when font is not unicode.